### PR TITLE
misc: Fixed small i18n discrepancies

### PR DESCRIPTION
**Pull request by fnurkla**

### CODE REVIEW

File: frontend/public/locales/sv/common.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "no_results": "Inga resultat",
   "regulations": "Reglemente",
   "statutes": "Stadgar",
-  "meetingDocuments": "Möteshandlingar",
+  "meetingDocuments": "Mötesdokument",
```
Comment (oliverlevay) [inline frontend/public/locales/sv/common.json:20]: "Möteshandlingar" is historically accurate https://www.dsek.se/arkiv/moteshandlingar/
image
and according to Google Translate the correct translation of "Meeting documents"

**Review by oliverlevay — CHANGES_REQUESTED** [review]
Nice work! We REALLY appreciate work like this, just one small change that I would like you to make.

File: frontend/public/locales/sv/common.json (hunk 1 above)
Comment (fnurkla) [reply frontend/public/locales/sv/common.json:20]: Svenska Akademien agrees with you, so I'll comply
image

**Review by oliverlevay — APPROVED** [review]
Great! Thank you !

**Pull request merged**